database/migrations/2023_09_09_150532_update_event_registration_add_remarks.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::table('event_registration', function ($table) {
+            $table->string('remarks', 1024)->nullable();
```

Comment:
For things like remarks, if we don't know the length we can also make this a text type so there is no length limit

Reply:
This was done very intentionally - a length limit was applied to the text area as well, and if you are going above 1 MiB of text in your remarks section, you should probably email the EC. I can remove this but I don't see any reason not to limit this to a reasonable length. 1024 characters is a _lot_ of remarks. Let me know if you still want me to remove this, and I can.

Reply:
Agree with Ian, however we already had this conversation and I think your logic makes sense. Now that we have the form input constrained with maxlength we should be ok.